101-cloud-shell-vnet-storage/azuredeploy.json

```diff
@@ -5,32 +5,34 @@
         "existingVNETName": {
             "type": "string",
             "metadata": {
-                "description": "Name of the virtual network to use for cloud shell containers"
+                "description": "Name of the virtual network to use for cloud shell containers."
             }
         },
         "existingStorageSubnetName": {
             "type": "string",
+            "defaultValue": "storagesubnet",
```

Comment:
remove the defaultValue - we don't want defaultValues here because this requires the actual value from the existing subnet - it's not likely the user has named the subnet the same way we have in the sample - so in practice this needs to be passed in... not defaulted.